Assets/Scripts/UISettings.cs

```diff
@@ -41,19 +41,44 @@ public void Deactivate()
 
    public void OpenSettingsFile()
    {
-      Process.Start(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", "CircuitCubed", "NotReaper", "NRConfig.txt"));
+      string FilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", Application.companyName, Application.productName, "NRConfig.txt");
+      string Arguments = "";
+
+      if ((Application.platform == RuntimePlatform.LinuxEditor) ^ (Application.platform == RuntimePlatform.LinuxPlayer))
+            FilePath = Path.Combine("file://" + Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "/.config/unity3d/" + Application.companyName + "/" + Application.productName + "/NRConfig.txt");
+            Arguments = "";
+
+      if ((Application.platform == RuntimePlatform.OSXEditor) ^ (Application.platform == RuntimePlatform.OSXPlayer)) {
+            FilePath = "open";
+            Arguments = Path.Combine(@"""" + Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "/Library/Application Support/" + Application.companyName + "/" + Application.productName + "/NRConfig.txt" + @"""");
+
+            if (Environment.OSVersion.Version.Major >= 18)
+                Arguments = Path.Combine(@"""" + Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "/Library/Application Support/" + Application.identifier + "/NRConfig.txt" + @"""");
+      }
+
+      Process.Start(FilePath, Arguments);
    }
 
+
    public void OpenSettingsFolder()
    {
-      ProcessStartInfo startInfo = new ProcessStartInfo
-      {
-         Arguments = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", "CircuitCubed", "NotReaper"),
-         FileName = "explorer.exe"
-      };
-      Process.Start(startInfo);
+      string Arguments = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", Application.companyName, Application.productName);
+      string FileName = "explorer.exe";
+
+      if ((Application.platform == RuntimePlatform.LinuxEditor) ^ (Application.platform == RuntimePlatform.LinuxPlayer)) {
```

Comment:
Is there a reason to XOR these instead of just ORing them?

Reply:
Completely unnecessary changed them all back to regular OR's